engine/class_modules/monk/sc_monk.cpp

```diff
@@ -1692,15 +1708,11 @@ struct charred_passions_t : base_action_t
       return;
 
     base_action_t::p()->proc.charred_passions->occur();
-    chp_damage->base_dd_min = chp_damage->base_dd_max = state->result_amount;
-    chp_damage->execute();
+    chp_damage->execute_on_target( state->target, state->result_amount );
```

Comment:
For reasons unbeknownst to all, the `execute_on_target` method can also handle setting `base_dd_min = base_dd_max = amt`. Previously, ChP would only ever hit the main target as well, oops!